cfpb/consumerfinance.gov
# Removed the `request.GET.form_field` and `request.POST.form_field` from Feedback form template.

* Satisfy CM
* Code Cleanup and Refactor (JS and JS-disabled)


## Additions

-


## Removals

-


## Changes

-


## How to test this PR
Run through steps in browser, once with JS enabled, and once with JS Disabled.

1. Load page with feedback form (`/ask-cfpb/a-box-on-my-credit-card-bill-says-that-i-will-pay-off-the-balance-in-three-years-if-i-pay-a-certain-amount-what-does-that-mean-do-i-have-to-pay-that-much-if-i-pay-that-much-and-make-new-purchases-will-i-still-owe-nothing-after-three-years-en-36/`)
2. Fill out form and submit.
3. Form should disappear showing success message.
4. Data should still be retrievable via Download Feedback


## Screenshots


## Notes and todos

-


## Checklist



  - PR titles are used to generate the change log in [releases](../../releases); good ones make that easier to scan.
  - Consider prefixing, e.g., "Mega Menu: fix layout bug", or "Docs: Update Docker installation instructions".
  - [This repo’s docs](https://cfpb.github.io/consumerfinance.gov/) (edit the files in the `/docs` folder) – for basic, close-to-the-code docs on working with this repo
  - CFGOV/platform wiki on GHE – for internal CFPB developer guidance
  - CFPB/hubcap wiki on GHE – for internal CFPB design and content guidance

### Front-end testing



#### Browser testing

Visually tested in the following supported browsers:



#### Accessibility


#### Other
